backend/main.go

```diff
@@ -35,31 +32,13 @@ func GoDotEnvVariable() ApplicationConfiguration {
 	}
 }
 
-//HomePage lets you get to the home page and notifies you with a println terminal.
-func HomePage(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprintf(w, "Welcome to the HomePage!")
-	fmt.Println("Endpoint Hit: homePage")
-}
-
-// FlutterInitRequest TODO: This is a test API for Flutter Integration, Temp.sends "Hello, World". Meta.
-func FlutterInitRequest(w http.ResponseWriter, r *http.Request) {
-	message := HelloWorld{Message: "Hello, World!"}
-
-	js, err := json.Marshal(message)
-	if err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
-	}
-	w.Header().Set("Content-Type", "application/json")
-	w.Write(js)
-}
-
 //HandleRequests listens for any requests that come in.
 func HandleRequests(applicationConfig ApplicationConfiguration) {
+	r := mux.NewRouter()
+	r = routes.ConnectNewsSubrouter(r)
+	// r.HandleFunc("/", FlutterInitRequest)
```

Comment:
Let's uncomment this til we properly integrate Flutter with the correct routes. This way, we can test.